Neos.Fusion/Classes/FusionObjects/ComponentImplementation.php

```diff
@@ -71,6 +72,17 @@ protected function getProps(array $context): \ArrayAccess
         return $props;
     }
 
+    protected function getComputedProps(array $context): ?\ArrayAccess
+    {
+        $this->runtime->pushContextArray($context);
+
+        $computed = $this->runtime->evaluate($this->path . '/__meta/computed<Neos.Fusion:ComponentComputedProps>');
```

Comment:
we need to play ping pong here and ask our friend `Neos.Fusion:ComponentComputedProps` to fetch all the `@computed` props, as the runtime doesnt give us access in the parent component about the subkeys